keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -207,20 +355,55 @@ promotion is an optional part of this KEP. As an intermediate solution we can
 also leave the package publishing on the Google side and focus on building them
 before graduating the KEP to GA.
 
-All architectures that are supported by the [package building tool][pkg-gen-kep] should be published.
-This KEP suggests to start with publishing a single supported architecture
-(e.g. `linux/amd64`) and extend that iteratively, when we verify that creating
-all packages for all architectures is fast enough to be done as part of the
-release process. If it turns out this step takes too long, we need to think
-about doing the package building & publishing asynchronous to the release
-process (see also: [Risks](#risks-and-mitigations)).
+At the time of writing this KEP, there are no Go libraries for working with OBS that we could use to integrate directly
+with `krel`. Eventually, we could evaluate if it makes sense to build such a library for our purposes. Until then,
+we'll use `osc` directly (by exec-ing), which also requires adding `osc` to our build images.
+
+[obs-rabbitmq]: https://rabbit.opensuse.org/
+
+#### Authentication to OBS and User Management
+
+The concept of API tokens in OBS is very limited and provides access only to a very few endpoints. In other words,
+it's not possible to use API tokens for publishing to OBS. Instead, we need to create some sort of a service account
+to be used when publishing packages. This is one time operation that can be done by SIG Release Leads.
+
+Users are managed manually via the OBS web interface. SIG Release Leads must have access to add/remove users from
+our OBS project. Release Managers should be given read/write access, so they can maintain and create projects and
+packages.
+
+#### How are packages used?
+
+The End Users can configure their systems’ package managers to use
+those different `${channel}`s of a kubernetes `${k8s_release}` for their
+corresponding package manager.
+
+A configuration for the package managers might look something like:
+
+- deb:
+  ```
+  # deb http://apt.kubernetes.io ${k8s_release} ${channel}
+  deb [signed-by=/etc/keyrings/kubernetes-keyring.gpg] http://obs.kubernetes.io/core:/stable:/v1.26/deb/ /
```

Comment:
We should not recommend legacy apt source formats anymore.

Use the deb822 formatted version:

```ini
Types: deb 
URIs: http://obs.kubernetes.io/core:/stable:/v1.26/deb/
Suites: ./
Signed-By: /etc/keyrings/kubernetes-keyring.gpg
```

Files have to end in `.sources` instead of `.list`

Reply:
@Conan-Kudo Does that work with older Ubuntu versions, e.g. 18.04 or 20.04?

Reply:
It was added in APT 1.1 (Ubuntu 16.04), so yes.

Reply:
I pushed a commit to address this comment.